release.yaml

```diff
@@ -106,3 +106,8 @@ rancher-gatekeeper-crd:
 - 100.0.1+up3.6.0
 system-upgrade-controller:
 - 100.0.2+up0.3.1
+- 100.0.2+up0.3.2
+sriov:
+- 100.0.2+up0.1.0
+sriov-crd:
+- 100.0.2+up0.1.0
```

Comment:
was this supposed to contain changes for sriov? confused about this commit, seems like it should be merged into the next one